### ISSUE TITLE
Accessibility: Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by angel1199410** [wbkd/react-flow#3273]
### Describe the Bug

While scanning my application using the aXe DevTools Accessibility plugin, it made me aware of the following bug:

> [aria-label attribute cannot be used on a g with no valid role attribute.](https://dequeuniversity.com/rules/axe/4.7/aria-allowed-attr?application=AxeChrome)

This was triggered specifically because I made the edges non-focusable, which removed the "role="button" attribute.

At minimum, being able to specify a role would be useful (or allowing us to spread generic HTML properties onto an edge/node.). Setting a role by default when `edgesFocusable={false}` is set could also be useful.

### Steps to Reproduce the Bug or Issue

This can be reproduced on any ReactFlow example by setting `edgesFocusabe={false}` and running the [aXe DevTools plugin](https://www.deque.com/axe/?utm_term=deque%20axe&utm_campaign=Search+-+axe+Pro+-+Branded).

### Expected behavior

The error should not appear.

### Screenshots or Videos

<img width="609" alt="Screenshot 2023-07-28 at 3 04 50 PM" src="https://github.com/wbkd/react-flow/assets/17990556/caa43c3b-7f70-4922-97b2-54b77154fb4a">

### Platform

- OS: macOS
- Browser: Chrome v115.0.5790.114
- Version: reactflow@^11.7.2

**Comment by moklick** [comment]
That's a good catch. Thanks! Would role="img" work in this case?

**Comment by moklick** [comment]
fixed in v11.8.0
